frontend/web/components/ExistingChangeRequestAlert.tsx

```diff
@@ -19,10 +19,11 @@ const ExistingChangeRequestAlert: FC<ExistingChangeRequestAlertType> = ({
     environmentId,
     feature_id: featureId,
   })
+  const date = useRef(moment().toISOString())
```

Comment:
Note that it was necessary (thanks to an assist from @kyle-ssg) to move this out to `useRef` to avoid an infinite recursion caused by the date being calculated on each re-render.